kubeadm/hack/publish-kubeproxy.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/Masterminds/semver"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+)
+
+const (
+	supportedVersions = ">= 1.17.0"
+)
+
+func dockerBuild(v *semver.Version, dockerfilePath string) error {
+	cmd := exec.Command("docker", "build", "--pull",
+		fmt.Sprintf("--build-arg=k8sVersion=v%s", v.String()),
+		fmt.Sprintf("--tag=sigwindowstools/kube-proxy:v%s", v.String()),
+		dockerfilePath,
+	)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
+}
+
+func dockerPush() error {
+	cmd := exec.Command("docker", "push", "sigwindowstools/kube-proxy")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
+}
+
+func main() {
+	dockerfilePath := os.Args[1]
```

Comment:
this will panic if the user did not feed the path as an argument.

better to have a:
```
if len(os.Args) < 2 {
	// print usage
}
```
even for completely scoped tools.

Reply:
Yeah, was just being lazy but I'll fix this